sdk/mongocluster/arm-mongocluster/src/static-helpers/pagingHelpers.ts

```diff
@@ -1,16 +1,88 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// Copyright (c) Microsoft Corporation.
```

Comment:
remove duplications here and create an issue in codegen to fix this.

Reply:
I think this should be a codegen bug, I add mongocluster in smoke and it still has this duplication